packages/core-forger/lib/client.js

```diff
@@ -39,6 +39,35 @@ module.exports = class Client {
     return response.data.success
   }
 
+  /**
+   * Emit the given event and payload to the local host.
+   * @param  {String} event
+   * @param  {Object} body
+   * @return {Object}
+   */
+  async emitEvent (event, body) {
+    // NOTE: Events need to be emitted to the localhost. If you need to trigger
+    // actions on a remote host based on events you should be using webhooks
+    // that get triggered by the events you wish to react to.
+
+    const host = this.hosts.find(host => host.includes('127.0.0.1'))
```

Comment:
it might be worth including all localhost IPs here:

Reply:
https://github.com/ArkEcosystem/core/blob/develop/packages/core/lib/config/devnet/plugins.js#L98